scrapy/extensions/feedexport.py

```diff
@@ -306,6 +357,16 @@ def _exporter_supported(self, format):
             return True
         logger.error("Unknown feed format: %(format)s", {'format': format})
 
+    def _batch_deliveries_supported(self, uri):
+        """
+        If FEED_STORAGE_BATCH_ITEM_COUNT setting is specified uri has to contain %(batch_time)s or %(batch_id)s
+        to distinguish different files of partial output
+        """
+        if self.storage_batch_size is None or '%(batch_time)s' in uri or '%(batch_id)s' in uri:
```

Comment:
To support `0` as well:

```suggestion
        if not self.storage_batch_size or '%(batch_time)s' in uri or '%(batch_id)s' in uri:
```